docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -98,10 +93,9 @@ prometheus.cephstorage-1    cephstorage-1.redhat.local  172.17.3.53:9092
 ----
 +
 [NOTE]
-After you migrate the monitoring stack, you lose High Availability: the monitoring stack daemons have no VIP and HAproxy anymore; Node exporters are still running on all the nodes: instead of using labels we keep the current approach as we want to not reduce the monitoring space covered.
+After you migrate the monitoring stack, you lose high availability. The monitoring stack daemons no longer have a Virtual IP address and HAProxy anymore. Node exporters are still running on all the nodes. Instead of using labels we keep the current approach as we want to not reduce the monitoring space covered.
 
```

Comment:
What is the current approach? I’m also unclear on what “reduce the monitoring space covered” means.

Reply:
I suggest to remove ". Instead of using labels we keep the current approach as we want to not reduce the monitoring space covered.", because it just confuses customers.